tools/crm_mon_print.c

```diff
@@ -946,6 +952,12 @@ print_status(pcmk__output_t *out, mon_output_format_t output_format,
     char *offline_nodes = NULL;
     char *offline_remote_nodes = NULL;
 
+    if (output_format == mon_output_console) {
+        print_opts |= pe_print_ncurses;
+    } else {
+        print_opts |= pe_print_printf;
+    }
```

Comment:
I believe this is unnecessary. pe_print_ncurses and pe_print_printf (along with pe_print_log, pe_print_html, pe_print_xml) should only be relevant to the print-based functions rather than the new output-based resource message. We just never dropped the relevant code from crm_mon when we finished converting it.

Reply:
Taking a quick random sampling, I believe you're correct.